addon-api/content-script/Tab.js

```diff
@@ -27,4 +40,16 @@ export default class Tab {
       return Promise.resolve();
     }
   }
+  /**
+   * @type {string} editor mode (or empty string for non-editors).
+   */
+  get editorMode() {
+    const pathname = location.pathname.toLowerCase();
+    const split = pathname.split("/").filter(Boolean);
+    if (!split[0] || split[0] !== "projects") return "";
```

Comment:
Might be better to return `null`, maybe?

Reply:
Note: I know I'm saying that after merging (lol), but no addon is using this yet (checking against empty string, or null) so it's fine.